### ISSUE TITLE
can etw file event support correlating filepath? 

### ISSUE THREAD

**Issue by yangjian123** [zodiacon/ProcMonXv2#2]
can ProcMonXv2 support correlating filepath  for etw fileio event, especially for filerenam event?

**Comment by zodiacon** [comment]
Yes, it's possible, by correlating the file unique key... but it requires work, which I didn't get around to yet :)
But I'm happy to receive PRs...

**Comment by yangjian123** [comment]
thanks for reply, but when will this feature be added, do you have a plan?

**Comment by zodiacon** [comment]
No specific plan, as I am currently juggling several projects. I'll see if I can plan for this in the near future.
